tcg_rim_extensions.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2020 Contributors to the Veraison project.
```

Comment:
Could this file go under extensions/ folder ?

Reply:
why do we need to create another package?

Reply:
All extensions (for future) can reside there. Is it not much neater rather than residing in the base package ?

Reply:
The separation is not that clear cut: one still needs to put the extension structures inside the base package structures, so it's not like you have two independent packages that you can decide whether to compose (or not), you always have to have both.